cloudinary-core/src/main/java/com/cloudinary/Url.java

```diff
@@ -417,6 +418,9 @@ public String generate(String source) {
             } catch (MalformedURLException ignored) {
             }
         }
+        if (AnalyticsUtils.token != null) {
+            url = (new StringBuilder()).append(url).append(AnalyticsUtils.analyticsPrefix).append(AnalyticsUtils.token).toString();
```

Comment:
The URL can already include a query param, if the URL already includes a query param, we should not append the analytics token.

Reply:
Fixing